### DISCUSSION TITLE
Missing information in the docu!

### DISCUSSION THREAD

**Discussion by DariusPlaczek** [pmndrs/react-spring#1520] [Support]
Hello, I have now discovered React Springt and have a question because the documentation doesn't really describe the problem. In the code "useTransition" there is a const [items, setItems] = useState(NUM_TRANS). 

What is NUM_TRANS? Is it an array, a string, number? 
Or what is written in the useState?

And where do you get config.molasses? Is it imported? It is not described anywhere on the page. 

https://react-spring.io/hooks/use-transition#you-can-transition-arrays

Thanks a lot

**Comment by joshuaellis** [comment]
It's the data you want to use to animate with. Typically I imagine it would be an array, but it could be a boolean or string and changing the item would therefore trigger an animation.

Configs, check out this https://react-spring.io/common/configs#presets

**Reply by DariusPlaczek** [reply]
When I fill constant items with [1,2,3,4] I get an error

**Reply by joshuaellis** [reply]
If you'd like some support, I'd recommend creating codesandbox, that way I can edit and show you 😄

**Reply by DariusPlaczek** [reply]
I thought that was the code for the numbers as shown in the preview. But apparently it is not :D
